src/actions/organization/organization_read.js

```diff
@@ -0,0 +1,15 @@
+const { Organization } = require('../../models');
+
+const readAction = async query => {
+  if (query) {
```

Comment:
Read action will receive only an ID. So this will be even simpler,

`const readAction = id => Organization.findById(id);`